pkgs/tools/security/clevis/default.nix

```diff
@@ -29,6 +30,8 @@ stdenv.mkDerivation rec {
     hash = "sha256-3J3ti/jRiv+p3eVvJD7u0ko28rPd8Gte0mCJaVaqyOs=";
   };
 
+  patches = [ ./tang-timeout.patch ];
```

Comment:
Why is this patch necessary and has there been an attempt to upstream it?

Reply:
The default timeout for tang is 300s, the patch reduces it to 10s for fallback on interactive unlocking. I can open a PR and pull the patch from the PR if needed.